src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexPersistence.cs

```diff
@@ -24,6 +24,7 @@ namespace Raven.Server.Documents.Indexes.Persistence.Corax;
 
 public sealed class CoraxIndexPersistence : IndexPersistenceBase
 {
+    private const bool DisableDictionaryTraining = false; // [DEBUG ONLY]: disable training.
```

Comment:
What did you use it for?

Reply:
We've similar thing during index compilation. It is used for disabling training. Pretty useful for debugging indexwriter flow since we do not have then two different index builder (trainer and indexer).

Reply:
OK